frontend/indigenousapprenticeship/.env.example

```diff
@@ -2,3 +2,4 @@ INDIGENOUSAPPRENTICESHIP_APP_URL="http://localhost:8000/indigenous-it-apprentice
 INDIGENOUSAPPRENTICESHIP_APP_DIR="/indigenous-it-apprentice"
 NODE_ENV=development
 API_URI="/graphql"
+APPLICATIONINSIGHTS_CONNECTION_STRING=""
```

Comment:
This variable does not have the "INDIGENOUSAPPRENTICESHIP" prefix.  If the connection string will be the same for the other apps then they will share the variable and this is OK.  If the others need to be different then you may want to add the prefix.

Reply:
This is going to be the same connection string for all the apps. So I leave it like that.